### ISSUE TITLE
Synchronous Upload of multiple images

### ISSUE THREAD

**Issue by komarovalexander** [ant-design/ant-design#47447]
### What problem does this feature solve?

we need to send files sequentially to the server, not in parallel 

### What does the proposed API look like?

synchronous?: boolean 
when it is true customRequest should be called sequentially, and called only after previous one upload was completed

**Comment by afc163** [comment]
Duplicate of #28950
